### PR TITLE
Add copy guidelines

### PR DESCRIPTION
**Pull request by joaoramos**
# Purpose of PR
Add copy guidelines to the website.

## PR Checklist
- [x] I have read the relevant `readme.md` file(s)
- [x] All commits follow our [Git commit message convention](https://github.com/contentful/forma-36/tree/master/packages/forma-36-react-components#commits)
- [x] Tests are added/updated/not required
- [x] Tests are passing
- [x] Storybook stories are added/updated/not required
- [x] Usage notes are added/updated/not required
- [x] Has been tested based on [Contentful's browser support](https://www.contentful.com/faq/about-contentful/#which-browsers-does-contentful-support)
- [x] Doesn't contain any sensitive information

## To-do
- [ ] Add circle cross icon @m10l 
- [ ] Create example cards @m10l 
- [ ] Add copy best practices to components @sarakalinoski 
- [ ] Remove suggested reading @m10l 
- [ ] Remove component examples @m10l 
- [ ] Update copy formatting based on suggestions @m10l 
- [x] Update copy index “for specific component copy check out components section” (with link) @sarakalinoski

### CODE REVIEW

**Comment by joaoramos** [comment]
I just changed base to be able to preview. I'll change back to `docWebsite` before merging.

**Comment by domarku** [comment]
Putting examples of DO's and DON'Ts as images makes them inaccessible and creates a bottleneck for future contributions. Would be better if we could implement it by using either

1. Bullet points with emoji, e.g. ✅ for DO's and 🚫 for DON'Ts
2. Card components with icons, like the ones in the images

**Comment by joaoramos** [comment]
> Putting examples of DO's and DON'Ts as images makes them inaccessible and creates a bottleneck for future contributions. Would be better if we could implement it by using either
> 
> 1. Bullet points with emoji, e.g. ✅ for DO's and 🚫 for DON'Ts
> 2. Card components with icons, like the ones in the images

That's a good idea 👏 How about [annotations like these](https://deploy-preview-159--forma-36.netlify.com/guidelines/copy/component-examples), though? @m10l What do we have to do to have components available in these pages?

**Comment by m10l** [comment]
> Putting examples of DO's and DON'Ts as images makes them inaccessible and creates a bottleneck for future contributions. Would be better if we could implement it by using either
> 
> 1. Bullet points with emoji, e.g. ✅ for DO's and 🚫 for DON'Ts
> 2. Card components with icons, like the ones in the images

Nice, I think using something text/component based would be a huge benefit for maintainability.

I vote for card components. After looking through the content, bullet points don't really make sense, as we're showing good/bad examples of a single point, rather than a list of good/bad points.

I'd suggest creating an `ExampleCard` component or similar as part of the documentation site, which could be used as follows:

**`capitalization.mdx`**
```mdx
# Capitalization

## Use sentence style caps

Sentence style caps, in which only the first word of a sentence or term is capitalized, should be used across the web app. This makes sentences a bit more scannable and our UX copy more approachable. Example uses:

- labels
- headings
- titles
- buttons
- menu items

<ExampleCardWrapper>
  <ExampleCard type="positive">
    View your website live
  </ExampleCard>

  <ExampleCard type="negative">
    View Your Website Live
  </ExampleCard>
</ExampleCardWrapper>


## Contentful UI elements are written in lowercase

Contentful UI elements should be written in lowercase consistently across the web app. Example uses:

- space
- entry
- content type
```

File: packages/forma-36-website/src/pages/guidelines/copy/grammar/capitalization/index.mdx [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Capitalization
+
+## Use sentence style caps
+
+Sentence style caps, in which only the first word of a sentence or term is capitalized, should be used across the web app.
+This makes sentences a bit more scannable and our UX copy more approachable.
+Example uses:
```
Comment (m10l) [inline packages/forma-36-website/src/pages/guidelines/copy/grammar/capitalization/index.mdx]: I'd change `Example uses:` to  `Places where you should use sentence style caps:` - makes it clearer what this list is.

File: packages/forma-36-website/src/pages/guidelines/copy/grammar/capitalization/index.mdx [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+- labels
```
Comment (m10l) [inline packages/forma-36-website/src/pages/guidelines/copy/grammar/capitalization/index.mdx]: What do we mean by labels, form labels? Maybe make it specific and remove any ambiguity.

File: packages/forma-36-website/src/pages/guidelines/copy/grammar/capitalization/index.mdx [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+- headings
+- titles
+- buttons
+- menu items
+
+![Build Status](/sentence-style-caps.jpg)
```
Comment (m10l) [inline packages/forma-36-website/src/pages/guidelines/copy/grammar/capitalization/index.mdx]: Copy and paste fail. If we stick with images, we should also write better alt text in general. This should be something like `Example of correct usage of sentence style caps - use "View your website live" rather than "View Your Website Live"`

File: packages/forma-36-website/src/pages/guidelines/copy/grammar/capitalization/index.mdx [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+## Contentful UI elements are written in lowercase
+
+Contentful UI elements should be written in lowercase consistently across the web app.
```
Comment (m10l) [inline packages/forma-36-website/src/pages/guidelines/copy/grammar/capitalization/index.mdx]: I'd remove `across the web app`

File: packages/forma-36-website/src/pages/guidelines/copy/grammar/index.mdx [hunk 5]
```diff
@@ -0,0 +1,3 @@
+# Grammar and mechanics
```
Comment (m10l) [inline packages/forma-36-website/src/pages/guidelines/copy/grammar/index.mdx]: I'd go with just `grammar`, or, make all of the directory names and urls match this title

File: packages/forma-36-website/src/pages/guidelines/copy/grammar/punctuation/index.mdx [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+
+Use periods on:
+
+- multiple sentences
+- any sentence followed by a link
+- modal body text
+
+![Use periods consistently](/use-periods-consistently-1.jpg)
+
+![Use periods consistently](/use-periods-consistently-2.jpg)
```
Comment (m10l) [inline packages/forma-36-website/src/pages/guidelines/copy/grammar/punctuation/index.mdx]: If we stick with images, I'd just put the example UI in the card and remove the text on the left. Also make sure the TextInput looks like a TextInput, again, removing ambiguity (I wasn't sure what this was at first).

File: packages/forma-36-website/src/pages/guidelines/copy/grammar/sentence-structure/index.mdx [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+
+## Focus on the objective of each sentence
+
+To make sentences scannable and easily understandable, start the sentence with the objective when possible.
+
+![Begin with the objective](/begin-with-objective.jpg)
+
+## Practice progressive disclosure
+
+To keep copy concise and actionable, eliminating extra details. Reveal details about a feature as the user interacts and requires more information.
```
Comment (m10l) [inline packages/forma-36-website/src/pages/guidelines/copy/grammar/sentence-structure/index.mdx]: > To keep copy concise and actionable, eliminating extra details.`

I think this should be `To keep copy concise and actionable, eliminate extra details.`

File: packages/forma-36-website/src/pages/guidelines/copy/suggested-reading/index.mdx [hunk 8]
```diff
@@ -0,0 +1,19 @@
+# Suggested reading
```
Comment (m10l) [inline packages/forma-36-website/src/pages/guidelines/copy/suggested-reading/index.mdx]: I'm not sure if this should be shared

File: packages/forma-36-website/src/pages/guidelines/copy/tone-of-voice/index.mdx [hunk 9]
```diff
@@ -0,0 +1,49 @@
+# Tone of voice
+
+Tone of voice is not just what you say, but how you say it. It's your choice of words, phrases and grammar. At Contentful, our brand voice never changes, but our tone may change.
+
+## Writing goals and principles
+
+With every piece of content we publish, we aim to:
```
Comment (m10l) [inline packages/forma-36-website/src/pages/guidelines/copy/tone-of-voice/index.mdx]: This doesn't make sense when applied to creating digital products

File: packages/forma-36-website/src/pages/guidelines/copy/grammar/punctuation/index.mdx [hunk 10]
```diff
@@ -0,0 +1,40 @@
+# Punctuation
+
+## Use periods consistently
+
+Keep UX copy scannable by eliminating unnecessary periods.
+Do not use periods on:
+
+- buttons
```
Comment (m10l) [inline packages/forma-36-website/src/pages/guidelines/copy/grammar/punctuation/index.mdx]: I feel like this would read better if it wasn't a list.

> **Do not** use periods on buttons, labels, tooltips, or bulleted lists. **Do** use periods when using multiple sentences, any sentence followed by a link, or modal body text.

File: packages/forma-36-website/src/pages/guidelines/copy/grammar/punctuation/index.mdx [hunk 11]
```diff
@@ -0,0 +1,40 @@
...
+- labels
+- tooltips
+- bulleted lists
```
Comment (m10l) [inline packages/forma-36-website/src/pages/guidelines/copy/grammar/punctuation/index.mdx]: This rule doesn't always make sense, like if the list item is a fully-formed sentence.

File: packages/forma-36-website/src/pages/guidelines/copy/grammar/punctuation/index.mdx [hunk 12]
```diff
@@ -0,0 +1,40 @@
...
+- any sentence followed by a link
+- modal body text
+
+![Use periods consistently](/use-periods-consistently-1.jpg)
+
+![Use periods consistently](/use-periods-consistently-2.jpg)
+
+## Avoid exclamation marks
+
+Exclamation marks may come across as shouting and do not match the brand tone established for the web app. There are some exceptions when it is appropriate to use exclamation marks, such as a greeting or congratulatory message.
```
Comment (m10l) [inline packages/forma-36-website/src/pages/guidelines/copy/grammar/punctuation/index.mdx]: Suggest rewording to not explicitly mention the web app but instead "Contentful products"

File: packages/forma-36-website/src/pages/guidelines/copy/grammar/capitalization/index.mdx [hunk 13]
```diff
@@ -0,0 +1,24 @@
+# Capitalization
+
+## Use sentence style caps
+
+Sentence style caps, in which only the first word of a sentence or term is capitalized, should be used across the web app.
```
Comment (m10l) [inline packages/forma-36-website/src/pages/guidelines/copy/grammar/capitalization/index.mdx]: I'd replace `across the web app` with `across Contentful products`

File: packages/forma-36-website/src/pages/guidelines/copy/grammar/tense/index.mdx [hunk 14]
```diff
@@ -0,0 +1,15 @@
+# Tense
+
+## Use present tense
+
+Our UX copy aims to enable users to navigate confidently across the web app. Keep UX copy actionable by using present tense for user interactions.
```
Comment (m10l) [inline packages/forma-36-website/src/pages/guidelines/copy/grammar/tense/index.mdx]: `across the web app` -> `across Contentful products`

File: packages/forma-36-website/src/pages/guidelines/copy/grammar/word-choice/index.mdx [hunk 15]
```diff
@@ -0,0 +1,46 @@
...
+
+![Use contractions when possible](/contractions.jpg)
+
+## Address users based on context (Your, My, We)
+
+Determine the appropriate pronoun based on the context. Focus on the user and the actions the user is taking whenever possible. Avoid mixing prounouns ("my" and "your”) in the same section— It can cause confusion to see both forms of addressing the user in the same context.
+
+### Second person (“you” or “your”)
+
+Use this style for most situations that require a pronoun (when the web app is speaking directly to the user).
```
Comment (m10l) [inline packages/forma-36-website/src/pages/guidelines/copy/grammar/word-choice/index.mdx]: `when the web app is speaking directly to the user` -> `when speaking directly to the user`

File: packages/forma-36-website/src/pages/guidelines/copy/grammar/word-choice/index.mdx [hunk 16]
```diff
@@ -0,0 +1,46 @@
...
+
+### First person (“my” or “I”)
+
+This may be used in a context that communicates the user's actions or ownership.
+
+![First person (I or my)](/first-person.jpg)
+
+### The pronouns “we” and “our”
+
+Overall, focus on the user rather than what you or your app is doing for the user. “We” should only be used in situations where the web app is taking an action for or on behalf of the user.
```
Comment (m10l) [inline packages/forma-36-website/src/pages/guidelines/copy/grammar/word-choice/index.mdx]: Not sure how to reword this, but again, remove web app I think. Here's a (not very good) suggestion:

`“We” should only be used in situations where the web app is taking an action for or on behalf of the user.` -> `“We” should only be used in situations when an action is being performed for or on behalf of the user.`

File: packages/forma-36-website/src/pages/guidelines/copy/index.mdx [hunk 17]
```diff
@@ -3,3 +3,12 @@ title: 'Guidelines / Copy Page'
 ---
 
 # Copy Guidelines
+
+## Welcome to the Contentful UX copy style guide
+
+The purpose of this guide is to empower the product team at Contentful to write consistent and intentional copy across our web app. We're happy to share it with other teams to adapt and learn from.
```
Comment (m10l) [inline packages/forma-36-website/src/pages/guidelines/copy/index.mdx]: `web app` -> `Contentful products`

File: packages/forma-36-website/src/pages/guidelines/copy/suggested-reading/index.mdx [hunk 18]
```diff
@@ -0,0 +1,19 @@
...
+
+We’re grateful to teams that have shared their process and approach through their design systems and copy style guides.
```
Comment (wichniowski) [inline packages/forma-36-website/src/pages/guidelines/copy/suggested-reading/index.mdx]: I also think this should be shared internally but not be part of our design system documentation.

Comment (joaoramos) [reply packages/forma-36-website/src/pages/guidelines/copy/suggested-reading/index.mdx]: I initially suggested the same, but Sara won me over with the argument that this shows we're open to the community and are clear about our inspiration and aspirations. There's also a couple other examples of design systems using it. I don't have a strong opinion.

File: packages/forma-36-website/src/pages/guidelines/copy/component-examples/index.mdx [hunk 19]
```diff
@@ -0,0 +1,16 @@
+# Component examples
```
[the remark this answers is not in the document]
Comment (domarku) [reply packages/forma-36-website/src/pages/guidelines/copy/component-examples/index.mdx]: +1

File: packages/forma-36-website/src/pages/guidelines/copy/suggested-reading/index.mdx (hunk 18 above)
Comment (m10l) [reply packages/forma-36-website/src/pages/guidelines/copy/suggested-reading/index.mdx]: I think it'd make sense to talk about these resources as part of a blog post, but I feel the design system itself should be focused on how to build a Contentful product, not how to build a design system.

File: packages/forma-36-website/src/pages/guidelines/copy/frequently-used-verbs/index.mdx [hunk 20]
```diff
@@ -2,83 +2,17 @@
...
-(Can often be replaced with “Set”).  
-EX: “Specify the color and quantity below.”
-
-## Submit
-
-Verb  
-To present or propose to another for review, consideration, or decision  
-(Consider context: This word has a formal tone and can be replaced with something lighter).  
-EX: “He submitted his resignation.”
+| Verb        | Type | Description                                                                                                                                                                                                  | Example                                 |
```
Comment (m10l) [inline packages/forma-36-website/src/pages/guidelines/copy/frequently-used-verbs/index.mdx]: Noice! Should change `verb` to something like `term/phrase/word`.

File: packages/forma-36-website/src/pages/guidelines/copy/tone-of-voice/index.mdx (hunk 9 above)
Comment (m10l) [reply packages/forma-36-website/src/pages/guidelines/copy/tone-of-voice/index.mdx]: > With all product copy, we aim to:

File: packages/forma-36-website/src/pages/guidelines/copy/component-examples/index.mdx (hunk 19 above)
[the remark this answers is not in the document]
Comment (sarakalinoski) [reply packages/forma-36-website/src/pages/guidelines/copy/component-examples/index.mdx]: I'd vote to keep grammar in sections to make this section more scannable. If someone wants to quickly look up a specific guideline, let's eliminate any extra work and put it where they expect it to be.

**Comment by sarakalinoski** [comment]
Screen Shot 2019-04-10 at 17 47 35

BUTTON COMPONENT

**Best practices**
- Position buttons consistently in the interface
- Reduce complexity by using a small number of actions. Too many actions can create confusion.

**Copy considerations**
- To make button action-oriented, use a verb. EX: "Add field", not "New field"
- Consider what happens after a user clicks on the button and communicate that clearly in the copy. 
 EX: “Get a UI Extension” vs “Explore all UI Extensions”  
- Use clear and succinct copy

**Comment by sarakalinoski** [comment]
Screen Shot 2019-04-10 at 17 51 38

**Best practices**
- Consider the context of where the note is displayed when selecting the type of note to use.
- Don't confuse notes with notifications, which appear as feedback after a user has performed an action.

**Copy considerations**
- Use clear and succinct copy
- Ensure note is scannable. Focus on the objective and limit the number of concepts in each sentence.
- Use progressive disclosure. Eliminate extra details that can be revealed as the user interacts and requires more information.
- Use active voice, present tense, and consider tone of voice depending on the circumstance.

**Comment by sarakalinoski** [comment]
Screen Shot 2019-04-10 at 17 55 19

No best practices

**Copy considerations**
- To make text link action-oriented, use a verb.
 EX: "Add field", not "New field"

- As a textlink sends a user away from the web app, clearly communicate what the user sees when they click on the link. EX: “View teams documentation” vs “Learn about teams”  

- Use clear and succinct copy

- If a text link is at the end of a sentence, don’t add period after the text link

**Comment by sarakalinoski** [comment]
No copy considerations for:
- Tables
- Typography

I'll start creating copy considerations for form components. Feel free to directly change anything above  as you see fit.

**Comment by m10l** [comment]
@sarakalinoski I've updated the component docs with your suggestions, I've changed the copy slightly to remove references to the web app
